packages/roosterjs-editor-api/lib/format/cacheGetHeaderLevel.ts

```diff
@@ -0,0 +1,16 @@
+import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
+import { Editor, cacheGetEventData } from 'roosterjs-editor-core';
+import { PluginEvent } from 'roosterjs-editor-types';
+
+const EVENTDATACACHE_HEADER_LEVEL = 'HeaderLevel';
+
+export default function cacheGetHeaderLevel(editor: Editor, event?: PluginEvent): number {
+    return cacheGetEventData<number>(event, EVENTDATACACHE_HEADER_LEVEL, () => {
+        for (let i = 1; i <= 6; i++) {
```

Comment:
Why check 6 levels? Any special reasons?

Reply:
It matches to HTML tag <h1> .. <h6>